app/models/dish.rb

```diff
@@ -0,0 +1,5 @@
+class Dish < ApplicationRecord
```

Comment:
Why do you choose this name? Will you only sell food or will you also serve drinks and other prospects?

Reply:
We're changing it to product and product_category